boards/arm/v2m_musca_s1/CMakeLists.txt

```diff
@@ -0,0 +1,65 @@
+#
+# Copyright (c) 2019-2020 Linaro Limited
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+zephyr_library()
+zephyr_library_sources(pinmux.c)
+zephyr_library_include_directories(${ZEPHYR_BASE}/drivers)
+
+if (CONFIG_BUILD_WITH_TFM)
+	# Set default image versions if not defined elsewhere
+	if (NOT DEFINED TFM_IMAGE_VERSION_S)
+		set(TFM_IMAGE_VERSION_S 0.0.0+0)
+	endif()
+
+	if (NOT DEFINED TFM_IMAGE_VERSION_NS)
+		set(TFM_IMAGE_VERSION_NS 0.0.0+0)
+	endif()
+
+	set(PREPROCESSED_FILE "${CMAKE_BINARY_DIR}/tfm/image_macros_preprocessed")
```

Comment:
`PREPROCESSED_FILE` and some other locations have changed in the latest TF-M update, see https://github.com/zephyrproject-rtos/zephyr/commit/310999da5b6c84f7842c7d797c68da96ebb72f01#diff-89eda17a4ad2615d313752b174e955c580154bf123e0d2de83262fa6270e9fae